README.md

```diff
@@ -32,7 +36,8 @@ for your learners to practice in.
     (If you do not have an account, you can quickly create one for free.)
     You must be logged in for the remaining steps to work.
 
-2.  In this page, click on the green "Use this template" button (top right)
+2.  In this page (<https://github.com/carpentries/workshop-template>),
```

Comment:
finding this button might warrant an image? when working with people unfamiliar with GitHub, the look to where the user profile is when I've said "top right"